guides/rails_guides/generator.rb

```diff
@@ -28,12 +28,12 @@
 #     enough:
 #
 #       # generates only association_basics.html
-#       ONLY=assoc ruby rails_guides.rb
+#       ONLY=assoc rake guides:generate
 #
 #     Separate many using commas:
 #
 #       # generates only association_basics.html and migrations.html
-#       ONLY=assoc,migrations ruby rails_guides.rb
+#       ONLY=assoc,migrations rake guides:generate
```

Comment:
can you change migrations to, say `configuring`, etc. file migrations doesn't exist now.